.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "test/spec"]
 	path = test/spec
 	url = https://github.com/WebAssembly/testsuite.git
-[submodule "test/experimental"]
-	path = test/experimental
+[submodule "experimental"]
```

Comment:
I had initially removed it, but check.py does `import experimental.buildbot.link_assembly_files as link_assembly_files` and needs it.